components/app/Calendar.vue

```diff
@@ -0,0 +1,185 @@
+    :key="group.name"
+    class="max-w-7xl mx-auto lg:pt-20 pt-10 px-4 relative"
+  >
+    <ProseH1 class="mb-8 text-center">
+      {{ group.name }}
+    </ProseH1>
+    <vue-cal
+      class="rounded-lg bg-white dark:bg-neutral-800 overflow-hidden shadow"
+      today-button
+      small
+      :events-on-month-view="true"
+      :twelve-hour="true"
+      :events="group.items"
+      :start-week-on-sunday="false"
+      :disable-views="['years', 'year', 'day']"
+      :time-from="8 * 60"
+      :time-to="22 * 60"
+      :time-step="60"
+      :on-event-click="onEventClick"
+    >
+      <template #arrow-prev>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-left"
+        />
+      </template>
+      <template #arrow-next>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-right"
+        />
+      </template>
+    </vue-cal>
+
+    <div
+      id="calendar-list-toggle"
+      class="w-[180px] bg-gray-400/20 rounded-lg absolute top-[158px] left-[22px]"
+    >
+      <div
+        class="w-1/2 inline-flex items-center"
+      >
+        <input
+          id="calendar-toggle"
+          name="calendar-list-toggle-radio"
+          type="radio"
+          class="hidden peer"
+          checked
+        >
+        <label
+          for="calendar-toggle"
+          class="w-full text-center px-3 py-1 cursor-pointer rounded-lg peer-checked:bg-primary peer-checked:text-white"
+        >
+          Calendar
+        </label>
+      </div>
+      <div
+        class="w-1/2 inline-flex items-center"
+      >
+        <input
+          id="list-toggle"
+          name="calendar-list-toggle-radio"
+          type="radio"
+          class="hidden peer"
+        >
+        <label
+          for="list-toggle"
+          class="w-full text-center px-3 py-1 cursor-pointer rounded-lg peer-checked:bg-primary peer-checked:text-white"
+        >
+          List
+        </label>
+      </div>
+    </div>
+
+    <AppModal
+      v-model="showEventModal"
+    >
+      <div class="flex items-center justify-between p-4 md:p-5 border-b rounded-t dark:border-gray-600">
+        <h3 class="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">
+          {{ selectedEvent.title }}
+        </h3>
+        <button
+          type="button"
+          class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm w-8 h-8 ms-auto inline-flex justify-center items-center dark:hover:bg-gray-600 dark:hover:text-white"
+        >
+          <Icon
+            class="w-6 h-6"
+            name="i-ph-x-light"
+            @click="showEventModal = false"
+          />
+          <span class="sr-only">Close dialog</span>
+        </button>
+      </div>
+      <div class="p-4 space-y-4">
+        <p class="text-sm text-gray-500">
+          {{ selectedEvent.start.format('DD/MM/YYYY') }}
+        </p>
+        <p
+          class="content-full"
+          v-html="selectedEvent.contentFull"
```

Comment:
I am not exactly sure what the entirety of contentFull has but I think it has the link inside it, correct?

We might want to but that other content variable here and then have the link below in an a tag so it is clickable :)

You might have already had plans to do this and I am jumping the gun :P Sorry

Reply:
Yeah definitely part of the plan! One issue I was having is sometimes the content from the calendar had HTML tags on the links, and sometimes they didn't. Also, some event descriptions had multiple links in them for the group/meetup, and then the event itself. So I'm not sure if there was someway we could standardize the calendar event to help grab the correct link for the event.

Or maybe it would be better to not worry about that and just make all links clickable, and forget about making a "view event" button, or something similar. I wasn't sure what the best approach would be. 😞

Reply:
I added a function to wrap all links in `<a>` tags to make them clickable. I also noticed some of the descriptions contain `**` and `*` wrapped words, so I added a simple markdown render function for those. 😄